### PR TITLE
Samples: fix samples for methods that already wait for jobs.

### PR DESCRIPTION
**Pull request by tswast**
This issue was discovered in https://github.com/googleapis/nodejs-bigquery/pull/66#discussion_r172368500

Fixing the promise logic uncovered the issue that the test data files could not be loaded
with the existing samples because a schema was not provided. Since Avro
files bundle a schema inline, I added an Avro test data file (identical
in content to data.json) so that these tests can run.

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by tswast** [comment]
Ping?

File: samples/tables.js [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 'use strict';
 
+var assert = require('assert');
```
Comment (jmdobry) [inline samples/tables.js]: s/`var`/`const`

File: samples/tables.js [hunk 2]
```diff
@@ -172,30 +174,24 @@ function copyTable(
...
-
   // Copies the table contents into another table
   bigquery
     .dataset(srcDatasetId)
     .table(srcTableId)
     .copy(bigquery.dataset(destDatasetId).table(destTableId))
     .then(results => {
-      job = results[0];
-      console.log(`Job ${job.id} started.`);
+      var job = results[0];
```
Comment (jmdobry) [inline samples/tables.js]: s/`var`/`const`/

File: samples/tables.js [hunk 3]
```diff
@@ -220,30 +216,24 @@ function loadLocalFile(datasetId, tableId, filename, projectId) {
...
-
   // Loads data from a local file into the table
   bigquery
     .dataset(datasetId)
     .table(tableId)
     .load(filename)
     .then(results => {
-      job = results[0];
-      console.log(`Job ${job.id} started.`);
+      var job = results[0];
```
Comment (jmdobry) [inline samples/tables.js]: s/`var`/`const`/

File: samples/tables.js [hunk 4]
```diff
@@ -274,30 +264,24 @@ function loadFileFromGCS(datasetId, tableId, bucketName, filename, projectId) {
...
-
   // Loads data from a Google Cloud Storage file into the table
   bigquery
     .dataset(datasetId)
     .table(tableId)
     .load(storage.bucket(bucketName).file(filename))
     .then(results => {
-      job = results[0];
-      console.log(`Job ${job.id} started.`);
+      var job = results[0];
```
Comment (jmdobry) [inline samples/tables.js]: s/`var`/`const`/

File: samples/tables.js [hunk 5]
```diff
@@ -334,30 +318,24 @@ function extractTableToGCS(
...
-
   // Exports data from the table into a Google Cloud Storage file
   bigquery
     .dataset(datasetId)
     .table(tableId)
     .extract(storage.bucket(bucketName).file(filename))
     .then(results => {
-      job = results[0];
-      console.log(`Job ${job.id} started.`);
+      var job = results[0];
```
Comment (jmdobry) [inline samples/tables.js]: s/`var`/`const`/

**Comment by tswast** [comment]
Could you merge too? I don't have permissions over here.

**Pull request merged**